apps/client/src/features/control/message/MessageControl.tsx

```diff
@@ -21,6 +25,30 @@ export default function MessageControl() {
         changeHandler={(newValue) => setMessage.presenterText(newValue)}
         actionHandler={() => setMessage.presenterVisible(!data.timerMessage.visible)}
       />
+      <div className={style.buttonSection}>
+        <label className={style.blinkingLabel}>Timer text blink</label>
+        <Button
+          className={style.blinkingButton}
+          variant={data.timerMessage.timerBlink ? 'ontime-filled' : 'ontime-subtle'}
+          leftIcon={
+            data.timerMessage.timerBlink ? <IoFlashlightOutline size='24px' /> : <IoFlashlightSharp size='24px' />
+          }
+          onClick={() => setMessage.timerBlink(!data.timerMessage.timerBlink)}
+          data-testid='toggle timer blink'
+        >
+          {/* {data?.onAir ? 'Timer text is blinking' : 'Timer text is solid'} */}
+        </Button>
+        <label className={style.blackoutLabel}>Blackout timer screens</label>
+        <Button
+          className={style.blackoutButton}
+          variant={data.timerMessage.timerBlackout ? 'ontime-filled' : 'ontime-subtle'}
+          leftIcon={data.timerMessage.timerBlackout ? <IoEyeOffOutline size='24px' /> : <IoEye size='24px' />}
+          onClick={() => setMessage.timerBlackout(!data.timerMessage.timerBlackout)}
+          data-testid='toggle timer blackout'
+        >
+          {/* {data?.onAir ? 'Ontime is On Air' : 'Ontime is Off Air'} */}
+        </Button>
```

Comment:
More nitpicking:

- can you remove commented code? you can then also use the self closing version of the elements like <Button />
- I think the blink button should also blink, to be SUPER annoying so that the operator doesn't forget it on
- How about the icon called `IoSunny` and `IoSunnyOutline` to make it blink?
- How about the icon called `IoMoon` for the blackout?

Reply:
on the sunny symbol i can aggree, for the blackout i actually like the eye symbol more if you dont mind

Reply:
I don't. Thank you for sticking up for what you think it's best